graph/mutators/comment.js

```diff
@@ -454,17 +456,20 @@ const createPublicComment = async (context, comment) => {
   // We then take the wordlist and the comment into consideration when
   // considering what status to assign the new comment, and resolve the new
   // status to set the comment to.
-  let actions = await resolveCommentModeration(context, comment);
+  let {actions, status} = await resolveCommentModeration(context, comment);
+
+  // Assign status to comment.
+  comment.status = status;
 
   // Then we actually create the comment with the new status.
-  comment = await createComment(context, comment);
+  const result = await createComment(context, comment);
```

Comment:
Why can't we just assign this to the original comment variable? We don't use the old one here, and truly, it's realized the new comment here instead.

Reply:
I think I added this because I used have more changes here. 
But I think it is good to have a different variable here.
Because  `comment` in my mind was the `input` an
`result` is the created comment. Different concept and as such I would assign them to different variables just to make it more clear.